src/test/resources/cucumber/features/integrationTests/cli/sastCliScan.feature

```diff
@@ -26,3 +27,10 @@ Feature: Cx-Flow CLI SAST Integration tests
       | filter-High-and-Medium | x+y             | 10          |
       | filter-only-Medium     | y               | 10          |
       | filter-invalid-cwe     | 0               | 0           |
+
+
+  Scenario: Testing SAST scans with bugTracker None
+    Given bug tracker is set to 'None'
+    When running cxflow to execute SAST scan
+    Then cxflow should not wait for scan results
+    And cxflow exit with success return code
```

Comment:
I think it's clearer to specify 0 as an argument. This also allows to reuse the validateExitCode method.

Reply:
changed to '0'